src/main/java/com/uber/cadence/worker/Worker.java

```diff
@@ -528,28 +597,49 @@ private String getStickyTaskListName() {
       private PollerOptions stickyWorkflowPollerOptions;
       private Scope metricScope;
 
+      /**
+       * When set to true it will create an affinity between the worker and the workflow run it's
+       * processing. Workers will cache workflows and will handle all decisions for that workflow
+       * instance until it's complete or evicted from the cache.
+       */
       public Builder setEnableStickyExecution(boolean enableStickyExecution) {
         this.enableStickyExecution = enableStickyExecution;
         return this;
       }
 
+      /**
+       * When Sticky execution is enabled this will set the maximum allowed number of workflows
+       * cached. This cache is shared by all workers created by the Factory.
+       */
       public Builder setCacheMaximumSize(int cacheMaximumSize) {
         this.cacheMaximumSize = cacheMaximumSize;
         return this;
       }
 
+      /**
+       * Maximum number of threads available for workflow execution across all workers created by
+       * the Factory.
+       */
```

Comment:
Document the default value.
Rename to set**M**axWorkflowThreadCount

Reply:
resolved